netlify.toml

```diff
@@ -0,0 +1,11 @@
+[context.deploy-preview]
+command = "hugo -b $DEPLOY_PRIME_URL"
+
+[context.deploy-preview.environment]
+HUGO_VERSION = "0.60"
```

Comment:
I would remove the specific version tag here, as we haven't pinned a version yet and are just using latest at the moment.  Either that or we can pin the version of `hugo-bin` in our `package.json`.

Reply:
it's pinned in our `yarn.lock` to that version

Reply:
buuut netlify doesn't seem to like that so I will remove